modules/workbench/modules/monitoring/modules/dashboards/assets/dashboards_json/distribution_metrics.json

```diff
@@ -3,102 +3,6 @@
-                "xyChart": {
-                    "dataSets": [
-                        {
-                            "timeSeriesQuery": {
-                                "timeSeriesFilter": {
-                                    "filter": "metric.type=\"logging.googleapis.com/user/workspace_operation_time\" resource.type=\"generic_node\" resource.label.\"namespace\"=\"${namespace}\"",
-                                    "aggregation": {
-                                        "perSeriesAligner": "ALIGN_DELTA",
-                                        "crossSeriesReducer": "REDUCE_SUM"
-                                    }
-                                },
-                                "unitOverride": "ms"
-                            },
-                            "plotType": "HEATMAP",
-                            "minAlignmentPeriod": "60s",
-                            "targetAxis": "Y1"
-                        }
-                    ],
-                    "timeshiftDuration": "0s",
-                    "yAxis": {
-                        "label": "y1Axis",
-                        "scale": "LINEAR"
-                    },
-                    "chartOptions": {
-                        "mode": "COLOR"
-                    }
-                }
-            },
-            {
-                "title": "Workspaces Performance",
-                "xyChart": {
-                    "dataSets": [
-                        {
-                            "timeSeriesQuery": {
-                                "timeSeriesFilter": {
-                                    "filter": "metric.type=\"logging.googleapis.com/user/workspace_operation_time\" resource.type=\"generic_node\" resource.label.\"namespace\"=\"${namespace}\"",
-                                    "aggregation": {
-                                        "perSeriesAligner": "ALIGN_SUM",
-                                        "crossSeriesReducer": "REDUCE_MEAN",
-                                        "groupByFields": [
-                                            "metric.label.\"operation_name\""
-                                        ]
-                                    }
-                                },
-                                "unitOverride": "ms"
-                            },
-                            "plotType": "LINE",
-                            "minAlignmentPeriod": "60s",
-                            "targetAxis": "Y1"
-                        }
-                    ],
-                    "timeshiftDuration": "0s",
-                    "yAxis": {
-                        "label": "y1Axis",
-                        "scale": "LINEAR"
-                    },
-                    "chartOptions": {
-                        "mode": "COLOR"
-                    }
-                }
-            },
             {
                 "title": "logging/user/api_method_time for test by label.method_name [99TH PERCENTILE]",
```

Comment:
Are we keeping this one because it uses the logging metrics and it still works?

Reply:
yes. I have that just because it's working. But now after checking https://github.com/all-of-us/workbench/blob/f0f6d8d2dfa9d159f4123ccfc4a64cee4f6d5c7e/api/src/main/java/org/pmiops/workbench/interceptors/RequestTimeMetricInterceptor.java#L75
and GCP console, shall we just remove the one above since we already have tracing and that metrics is not used?

Reply:
I don't really have an opinion here

Reply:
I might just drop it for now as that's the only one used in API server...